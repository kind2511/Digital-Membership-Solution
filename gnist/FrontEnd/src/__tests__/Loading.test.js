import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../components/Loading';

test('Loading component snapshot', () => {
  const component = renderer.create(<Loading />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
