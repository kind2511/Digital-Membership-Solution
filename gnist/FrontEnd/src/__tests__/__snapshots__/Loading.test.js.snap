// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component snapshot 1`] = `
<div
  className="loading-container"
>
  <div
    className="loading-spinner"
  />
  <p>
    Loading...
  </p>
</div>
`;
